pkg/apply/apply.go

```diff
@@ -86,9 +102,53 @@ func ApplyObject(ctx context.Context, client *cnoclient.Client, obj Object, subc
 		log.Printf("could not encode %s for apply", objDesc)
 		return fmt.Errorf("could not encode for patching: %w", err)
 	}
-	if _, err := client.Dynamic().Resource(rm.Resource).Namespace(namespace).Patch(ctx, name, types.ApplyPatchType, data, patchOptions); err != nil {
+	if _, err := clusterClient.Dynamic().Resource(rm.Resource).Namespace(namespace).Patch(ctx, name, types.ApplyPatchType, data, patchOptions); err != nil {
 		return fmt.Errorf("failed to apply / update %s: %w", objDesc, err)
 	}
 	log.Printf("Apply / Create of %s was successful", objDesc)
 	return nil
 }
+
+func getObjectCopy(ctx context.Context, obj Object, client *cnoclient.Client) (Object, error) {
+	anno := obj.GetAnnotations()
+	if anno == nil {
+		return nil, fmt.Errorf("no annotations defined")
+	}
+
+	parts := strings.Split(anno[names.CopyFromAnnotation], "/")
```

Comment:
Will this work for non-namespaced objects? Is it just `defaut//CoolRoleBinding` ?

Reply:
yes, I was able to verify that it works